### PR TITLE
Added web element action methods

### PR DESCRIPTION
**Pull request by DanteUkraine**

### CODE REVIEW

File: src/web.element.ts [hunk 1]
```diff
@@ -1,14 +1,51 @@
-import { FrameLocator, Locator, Page } from "playwright-core";
-import { cloneDeep } from "lodash";
-import { BrowserInstance } from "./browser";
-import { expect } from "@playwright/test";
+import {FrameLocator, Locator, LocatorScreenshotOptions, Page} from 'playwright-core';
```
Comment (vromants) [inline src/web.element.ts:1]: no spaces

File: src/web.element.ts [hunk 2]
```diff
@@ -1,14 +1,51 @@
...
+type SelectTextOptions = Parameters<Locator['selectText']>[0];
+type SetCheckedOptions = Parameters<Locator['setChecked']>[1];
+type SetInputFilesType = Parameters<Locator['setInputFiles']>[0];
+type SetInputFilesOptions = Parameters<Locator['setInputFiles']>[1];
+type TapOptions = Parameters<Locator['tap']>[0];
+type TextContentOptions = Parameters<Locator['textContent']>[0];
+type TypeOptions = Parameters<Locator['type']>[1];
+type UncheckOptions = Parameters<Locator['uncheck']>[0];
+type WaitForOptions = Parameters<Locator['waitFor']>[0];
+///////////////////////////////////////////////////////////////
```
Comment (vromants) [inline src/web.element.ts:48]: what is intention of that line ?

File: test/browser.spec.ts [hunk 3]
```diff
@@ -1,12 +1,9 @@
-import {BrowserInstance, BrowserName} from "../src";
-import chai, {expect} from "chai";
-import {afterEach, test} from "mocha";
-import {webkit} from "playwright-core";
+import {BrowserInstance, BrowserName} from '../src';
```
Comment (vromants) [inline test/browser.spec.ts:1]: no spaces between braces

File: test/browser.spec.ts [hunk 4]
```diff
@@ -1,12 +1,9 @@
...
+import chai, {expect} from 'chai';
+import {afterEach, test} from 'mocha';
+import {webkit} from 'playwright-core';
 import chaiAsPromised from 'chai-as-promised'
-import * as path from 'path';
-
-const localFilePath = `file://${__dirname.replace(/\//g, path.sep)}/test.html`;
-
+import {localFilePath} from './utils'
```
Comment (vromants) [inline test/browser.spec.ts:6]: no spaces between braces

**Review by vromants — CHANGES_REQUESTED** [review]

**Review by vromants — APPROVED** [review]

**Review by v-romant — APPROVED** [review]

**Pull request merged**